feature/lldp/tests/core_lldp_tlv_population_test/core_lldp_tlv_population_test.go

```diff
@@ -114,10 +114,13 @@ func verifyNodeConfig(t *testing.T, nodeTelemetry *device.DevicePath, port *onda
 	} else {
 		t.Errorf("LLDP SystemName is not proper, got %s", state.GetSystemName())
 	}
-	if conf.GetInterface(port.Name()).GetName() != state.GetInterface(port.Name()).GetName() {
-		t.Errorf("LLDP interfaces/interface/state/name got: %s, want: %s.", state.GetInterface(port.Name()).GetName(),
-			conf.GetInterface(port.Name()).GetName())
+	if lldpEnabled {
```

Comment:
this nested conditional can be combined to reduce indentation: https://google.github.io/styleguide/go/decisions#conditionals-and-loops

it appears short circuiting is not required here so you can do something like the following:

```
stateMatch := conf.GetInterface(port.Name()).GetName() == state.GetInterface(port.Name()).GetName()
if lldpEnabled && !stateMatch {
  t.Errorf()
}
```

an alternative that may reduce line length even further is:
```
got := state.GetInterface(port.Name()).GetName()
want := conf.GetInterface(port.Name()).GetName()
if lldpEnabled && got != want {
  t.Errorf("LLDP interfaces/interface/state/name = %s, want %s", got, want)
}
```

Reply:
Hi @alexmasi , Thanks for reviewing. Made changes accordingly. Please take a look.